pyproject.toml

```
@@ -29,7 +29,7 @@ dependencies = [
     "prtpy",
     "pydot",
     "dataclasses-json",
-    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@3.0.0.dev11",
+    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@fix/issue_191",
```

This will be changed accordingly after approval:

```suggestion
    "sdx-datamodel @ git+https://github.com/atlanticwave-sdx/datamodel@3.0.0.dev12",
```